server/routes/shared_api/place.py

```diff
@@ -330,7 +330,7 @@ def child_fetch(parent_dcid):
   place_dcids = place_dcids + overlaps_response.get(parent_dcid, [])
 
   # Filter by wanted place types
-  place_type = get_place_type(parent_dcid)
+  place_type = place_type(parent_dcid)
```

Comment:
Can you rename this to `parent_place_type` or something like that? A little confusing to overwrite the name `place_type`.